build.sh

```diff
@@ -345,11 +355,15 @@ function tag_release {
 }
 
 function push_branches {
-	local UNIQUE_REFS=$(echo $@ | tr ' ' '\n' | sort -u | tr '\n' ' ')
+	if [[ -z "$DRY_RUN" ]]
+	then
+		local UNIQUE_REFS=$(echo $@ | tr ' ' '\n' | sort -u | tr '\n' ' ')
 
-	msg_info -n "Pushing branches: "
-	git push origin $UNIQUE_REFS
-	msg_warn "Skipping"
+		msg_info -n "Pushing branches: "
+		git push origin $UNIQUE_REFS
+	else
+		msg_warn "Skipping push"
```

Comment:
In fact, I was thinking `git log -p` would be awesome in the dry run case, so I'll amend with that.  :+1: